tools/terraform/dev/cosmosdb.tf

```diff
@@ -0,0 +1,22 @@
+# Not possible to create a collection - only up to database
```

Comment:
How about removing this file if we don't use this file. 😁

Reply:
Good point, removed 2 empty files. QnAMaker and ComsosDB can not be created like we expected.